### ISSUE TITLE
AAR file missing mandatory res/ folder

### ISSUE THREAD

**Issue by nudge** [paypal/PayPal-Android-SDK#383]
### Required Information

- PayPal Android SDK Version:
2.15.1

### Issue Description
The packaged AAR file is missing the `/res/` folder. This causes build tools (like OkBuck) to fail when building.
According to the Android Developer guidelines, the `/res/` is mandatory in AAR files.
Ref: https://developer.android.com/studio/projects/android-library.html#aar-contents

Manually adding an empty `/res/` folder into the AAR works locally.

Error message from OkBuck for reference:
```
ERROR: resource directory 'buck-out/bin/.okbuck/cache/__unpack_com.paypal.sdk.paypal-android-sdk-2.15.1.aar#aar_unzip__/res' does not exist
```

If this could be addressed in future releases that would be appreciated.

**Comment by lkorth** [comment]
@nudge we'll get this sorted out. We'll probably just end up distributing a jar since there are no resources, but at a minimum we'll make sure there's a res folder. Thanks for bringing this to our attention.

**Comment by jaypatel512** [comment]
Fixed in v2.15.2.

**Comment by nudge** [comment]
Thanks guys, appreciated!
